cmd/aws-sigv4-proxy/main.go

```diff
@@ -138,6 +141,10 @@ func shouldLogSigning() bool {
 	return *logSinging || *debug
 }
 
+func useUnsignedPayload() bool {
```

Comment:
I don't think we need this helper function because the command line parser would set the default to "false" as specified in line 51.

Reply:
Removed and just using the variable as the default is handled at the flag level.